pom.xml

```diff
@@ -1600,6 +1600,19 @@
     </profile>
 
     <profile>
+      <id>hadoop-2.6</id>
+      <properties>
+        <hadoop.version>2.6.0</hadoop.version>
```

Comment:
Hm, but why does that mean a separate Hadoop profile is needed? it sounds like you're saying this change depends on Hadoop 2.6? Let's ask Jenkins to find out.